backend/lib/endpoints/auth.js

```diff
@@ -135,6 +136,31 @@ async function routes(app) {
       throw app.httpErrors.internalServerError();
     }
   });
+
+  app.post(
+    "/change-password",
+    { preHandler: [app.getServerToken], schema: changePasswordSchema },
+    async (req) => {
+      const { body, token } = req;
+      const { email } = body;
+
+      try {
+        const responseMessage = await Auth0.changePassword(token, email);
+        req.log.info(
+          `Change password email created successfully for email=${email}`,
+        );
+        return { email, responseMessage, token };
```

Comment:
Should the token also be returned here?

Reply:
No the server token cannot be returned to the frontend because this can do everything with the Auth0  management API.
Only thing that can be returned is the access token JWT with authorization scopes for the signup & login, 
But I think in fact we should probably change that for Id Token instead eventually, I didn't add this originally but I think the access token also contains scopes for the Auth0 API and not only for use in our API.

Reply:
makes perfect sense. Thanks!